Public/Src/FrontEnd/Script/Modules/ModuleRegistry.cs

```diff
@@ -23,6 +24,12 @@ public sealed class ModuleRegistry : IModuleRegistry
         /// </summary>
         public GlobalModuleLiteral GlobalLiteral { get; }
 
+        /// <nodoc />
+        public PrimitiveTypes PrimitiveTypes { get; }
+

```

Comment:
just checking. By doing this type of things you may be extending the lifespan of the containing instance. In the past the module registry was confined to AST/evaluation only. Is that still the case?

Reply:
I checked for the app, if anything they globals and primitives are initialized later now. They should all be freed at the same time as before